guide.md

```diff
@@ -0,0 +1,406 @@
+Getting Started 2.0
+===
+
+For this guide, we're going to walk through some simple examples for each of the
+pieces of CRUD. Each step in this guide will build on the previous, and is meant
+to be followed along. This guide assumes that you're using PostgreSQL on
+Rust nightly. We'll talk about how to use Diesel on stable Rust in the final
+chapter. Before we start, make sure you have PostgreSQL installed and running.
+
+The first thing we need to do is generate our project.
+
+```shell
+cargo new diesel_demo
+cd diesel_demo
+```
+
+First, let's add Diesel to our dependencies. We're also going to use a tool
+called [`.env`][dotenv-rust] to manage our environment variables for us. We'll
+add it to our dependencies as well.
+
+[dotenv-rust]: https://github.com/slapresta/rust-dotenv
+
```

Comment:
Yes, this will go into those little terminal windows with the file name in the top bar